doc/classes/VisualShaderNodeReroute.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="VisualShaderNodeReroute" inherits="VisualShaderNode" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A node that allows rerouting a connection within the visual shader graph.
+	</brief_description>
+	<description>
+		Automatically adapts its port type to the type of the incoming connection and ensures valid connections.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="get_port_type" qualifiers="const">
+			<return type="int" enum="VisualShaderNode.PortType" />
+			<description>
+				Returns the port type of the reroute node.
+			</description>
+		</method>
+		<method name="set_port_type">
+			<return type="void" />
+			<param index="0" name="port_type" type="int" enum="VisualShaderNode.PortType" />
+			<description>
+				Sets the port type of the reroute node. This should not be called manually since it is automatically set by the [VisualShader].
```

Comment:
If it shouldn't be called manually then why is it exposed? You can prefix a method with `_` to hide it.

Reply:
I didn't want to expose it and actually forgot how you hide a method :)